internal/export/config.go

```diff
@@ -47,6 +47,8 @@ type Config struct {
 	TruncateWindow time.Duration `envconfig:"TRUNCATE_WINDOW" default:"1h"`
 	MinWindowAge   time.Duration `envconfig:"MIN_WINDOW_AGE" default:"2h"`
 	TTL            time.Duration `envconfig:"CLEANUP_TTL" default:"336h"`
+	// The worker considers export batches closed before the now() minus BatchCloseTimeShift minutes ago.
```

Comment:
I have a hard time comprehending this comment. I am not entirely sure my recommendation here is any better though.

```suggestion
	// The worker considers export batches closed during the last BatchCloseTimeShift minutes.
```